### DISCUSSION TITLE
Medusa.js server stops responding after a while

### DISCUSSION THREAD

**Discussion by vholik** [medusajs/medusa#5680] [Q&A]
We faced this issue on a few projects as well so I don't think that is some code issue. The problem is that once in a while (1 or 2 times in the day) the server stops responding to the incoming requests and doesn't even give timeouts. We have Checkly installed so when server stops responding we should restart the app and the problem will be resolved for some time. Can anybody recommend some tool to debug this problem or guide to solution. Thanks in advance.

**Comment by srindom** [comment]
A few questions that would be helpful to get answers to:
- Are the other projects your are referring to also Medusa projects? 
- Have you observed any patterns around when this happens?
- Are you using the Redis event bus?
